app/views/home/about.html.erb

```diff
@@ -341,6 +342,22 @@
           <p>Astronomer exploring the role of data science in academia at <a href="http://escience.washington.edu/">University of Washington's eScience Institute</a>. Core contributor to <a href="http://scikit-learn.org/">scikit-learn</a> and other science-focused Python packages.</p>
         </div>
 
+        <div class="main" id="business_model">
+          <h1>Business Model</h1>
+          <p>The Journal of Open Source Software is an open access journal <em>committed to running at minimal costs, with zero publication fees (<a href="https://en.wikipedia.org/wiki/Article_processing_charge">article processing charges</a>) or subscription fees</em>.
+            With volunteer effort from our editorial board and community reviewers and minimal infrastructure costs we believe JOSS can remain a free community service.</p>
+
```

Comment:
The costs are minimal, but there are some expenses. Should we add that donations also keep the cost free for authors?

e.g. 'With volunteer effort from our editorial board, community reviewers, donations and minimal infrastructure costs ... '